server/routes/visitors.ts

```diff
@@ -18,6 +17,55 @@ const getSelectedSlot = (slotsList: VisitSlotList, selectedSlot: string): VisitS
     .find(slot => slot.id === selectedSlot)
 }
 
+const checkSession = ({
+  stage,
+  visitData,
+  res,
+}: {
+  stage: number
+  visitData: VisitSessionData
+  res: Response
+  // eslint-disable-next-line consistent-return
+}): void => {
+  if (!visitData) {
+    return res.redirect('/search/')
```

Comment:
Not necessary for now, but I think something before the redirects like `req.flash('errors', [{ msg: 'Session expiry/error message...' }]` would be picked up if there was the errorSummary component included on the target page (and it had `{ errors: req.flash('errors') }`) in the context passed to view.

Reply:
This was an MVP as we discussed so we can always alter it. Basically if there's an error with the session it'll just throw you back to the point it can do.